JASP-Desktop/html/js/utils.js

```diff
@@ -79,6 +79,7 @@ function formatColumn(column, type, format, alignNumbers, combine, modelFootnote
             if (window.globSet.pExact) {
                 sf = 4;
 			} else {
+				if (dp < 3) dp = 3
```

Comment:
If you fix the number of decimals to values 3+ then this will keep getting smaller (e.g., `< 0.00001`). Do we want to go "exact" unless `< 0.001` or do we also want to show the `< 0.05` and `< 0.01` @AlexanderLyNL?)

Reply:
If I look at 0.11.1 we didn't used to do that either. Which is definitely a good thing.

So I guess the rules are:

If "Display exact p-values" is **off**:
if value < 0.001 ---> `< 0.001` (**does not** follow the "Fix the number of decimals" setting)
if value >= 0.001 ---> value (**does** follow the "Fix the number of decimals" setting)

If "Display exact p-values" is **on**:
if value < 0.001 ---> value (**does** follow the "Fix the number of decimals" setting)
if value >= 0.001 ---> value (**does** follow the "Fix the number of decimals" setting)

Reply:
Yes correct:

> If "Display exact p-values" is **off**:
> if value < 0.001 ---> `< 0.001` (**does not** follow the "Fix the number of decimals" setting)
> - if value >= 0.001 ---> value (**does** follow the "Fix the number of decimals" setting)
- if value is in between (0.001, 0.01), then `< 0.01`
- if value is in between(0.01, 0.05), then `< 0.05`
- if value is larger than 0.05, then `value` follow the "Fix the number of decimals" setting

Reply:
Are we sure we want `< 0.01` and `< 0.05`? At least 0.05 is not something we ever did and it's also not a very convincing value to report like that. (Not sure about 0.01 I didn't test that)

Reply:
The rules are:
If "Display exact p-values" is off:
     if number of decimals is < 3, then number of decimals is 3 so that
          if value < 0.001 (eg. 0.000406) --> < .001 
          if value > 0.001 (eg. 0.00406)   --> .004
     if number of decimals is 4 (for example)
          if value < 0.0001 --> < .0001 
          if value > 0.0001 --> value with 4 decimals
If "Display exact p-values" is on:
it takes always the specified number of decimals. So it it 2
     if value < 0.01 (eg. 0.00406) -> 4.06e-3
     if value > 0.01 (eg. 0.0406) -> 0.04

Reply:
No we don't want `.0001`, `0.001` is the lowest we go

Reply:
OK. I put the situation back to before https://github.com/jasp-stats/jasp-desktop/pull/4065, and I solve the issue (https://github.com/jasp-stats/jasp-issues/issues/706) by setting a scientific notation when the p-value is bigger than 0.001 but needs more than the 'number of decimals' (for example: number of decimals is 2, and p-value is 0.00412).

That means that when exact p-value is off:

- value < 0.001 is always written with '< .001' (does not depend on number of decimals)
- value >= 0.001
if 'number of decimals' >= 3: value is written always in decimal format
if 'number of decimals' = 2 (or 1):
if value < 0.01 then value is written in scientific format
if value >= 0.01 then value is written in decimal format